framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/OpenShiftConstants.java

```diff
@@ -29,7 +29,9 @@
 
     public static final String KIE_APP_SECRET = "kie.app.secret";
     public static final String KIE_IMAGE_STREAMS = "kie.image.streams";
-    public static final String KIE_APP_TEMPLATE = "kie.app.template";
+    public static final String KIE_APP_TEMPLATE_WORKBENCH_KIE_SERVER_DATABASE = "kie.app.template.workbench.kie-server.database";
```

Comment:
What type of thing do these 3 properties point to? Is that some url, some local file?

I'm kind of concerned that we're encoding a log of information in plain Strings here. Bunch of properties for which it's not clear what should I set as a value for them. Can we at least have this documented in javadocs?